webapp-django/crashstats/symbols/tests/test_views.py

```diff
@@ -67,6 +69,7 @@ def mocked_get_key(key_name):
                 if key_name in self.known_bucket_keys:
                     mocked_key = mock.Mock()
                     mocked_key.key = key_name
+                    mocked_key.content_type = 'application/binary-octet-stream'
```

Comment:
Now I don't understand. This is not testing the `key.content_type = settings.SYMBOLS_MIME_OVERRIDES[ext]` line, is it?